src/Npgsql/NpgsqlConnector.cs

```diff
@@ -2008,6 +2009,11 @@ void HandleParameterStatus(string name, string value)
         }
 
         #endregion Misc
+
+        public IDictionary<string, long> RetrieveStatistics()
```

Comment:
nit: Add doc comments (summary, params, returns) to members as you go. It makes it easier to step in and review.